README.md

```diff
@@ -228,7 +228,7 @@ The final result should look something like this:
 ## 5) Digging Deeper
 
 ### Scopes
-So far, we've only been accessing public data. If you'd like to access private data for things like private emails, repositories or organization information, You'll need to request specific access to those items using [Scopes](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/). It's also possible to reqeust additional permissions from a user at a later point in time. After adding additional permissions to your OAuth Application, your users will be prompted to approve the permission upgrade.
+So far, we've only been accessing public data. If you'd like to access private data for things like private emails, repositories or organization information, You'll need to request specific access to those items using [Scopes](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/). It's also possible to reqeust additional permissions from a user at a later point in time. After adding additional permissions to your OAuth Application, your users will be prompted to approve the permission upgrade before you can access any additional user data from that user.
 
 ### Persisting Access Tokens
-Currently, users are required to 'Sign In with GitHub' every time they visit your OAuth Application. It's possible to persist their Access Token to maintain their session as long as they're  signed into GitHub. To do that, You'll need to save their Access Token in a secure database. 
+Currently, users are required to **Sign In with GitHub** every time they visit your OAuth Application. It's possible to persist their access token to maintain their session as long as they're signed into GitHub. To do that, you'll need to save their access token to a database [DEGW: How should we handle this? This is a topic too big for this quickstart. Can we point them somewhere? We don't want to leave them hanging!!]
```

Comment:
The plan for the 'Next Steps' section here is to illustrate where using an OAuth app is a little bit of a headache, and how building with a GitHub App makes things easier.